spirv_cross/src/compiler.rs

```diff
@@ -23,6 +23,14 @@ impl spirv::ExecutionModel {
     }
 }
 
+impl spirv::Decoration {
+    fn as_raw(&self) -> spv::Decoration {
+        match *self {
+            spirv::Decoration::DescriptorSet => spv::Decoration::DecorationDescriptorSet,
```

Comment:
Were the other enum types out intentionally? I'm hoping we can [simplify the enum mappings somehow](https://github.com/rust-lang-nursery/rust-bindgen/issues/777) in the future.

Reply:
I currently don't need them so I didn't bother adding them, sry for my laziness 😊